instrumentation/jaxrs-client/jaxrs-client-1.1/src/main/java/io/opentelemetry/auto/instrumentation/jaxrsclient/v1_1/JaxRsClientV1Instrumentation.java

```diff
@@ -108,9 +96,7 @@ public static void onExit(
         @Advice.Thrown final Throwable throwable,
         @Advice.Local("otelSpan") Span span,
         @Advice.Local("otelScope") Scope scope) {
-      if (scope != null) {
-        scope.close();
-      }
+      scope.close();
```

Comment:
i think still need the `scope != null` here, until we do https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/912#issuecomment-673023737